src/containerapp/azext_containerapp/_utils.py

```diff
@@ -1610,8 +1610,8 @@ def get_default_workload_profiles(cmd, location):
 
 
 def ensure_workload_profile_supported(cmd, env_name, env_rg, workload_profile_name, managed_env_info):
-    profile_names = [p["name"] for p in safe_get(managed_env_info, "properties", "workloadProfiles", default=[])]
-    if workload_profile_name not in profile_names:
+    profile_names = [p["name"].lower() for p in safe_get(managed_env_info, "properties", "workloadProfiles", default=[])]
```

Comment:
Store/display names in case-sensitive format. Comparisons need to be case-**in**sensitive

```suggestion
    profile_names = [p["name"] for p in safe_get(managed_env_info, "properties", "workloadProfiles", default=[])]
```

Reply:
For matching the profiles against a list, shouldn't we do this in a case insensitive manner?

Reply:
yes, my suggestion is to have it in original format to display in error message.